joss/paper.md

```diff
@@ -38,7 +38,7 @@ bibliography: paper.bib
 
 # Summary
 
-The current cosmological model $\Lambda$CDM is more and more challenged by a number of observations of the Universe. In particular, observations seem to favor different properties of dark matter and dark energy than previously thought. Strong Gravitational Lensing (SGL) is one of the best observational tools to address these challenges, because it reveals the distribution of all the matter (luminous or not) and can be used to constrain the properties of dark energy.
+Any mass concentration in the Universe, luminous or dark, from vast galaxy clusters to stars within galaxies, can be studied through its gravitationally induced deflections of light rays from background sources – a phenomenon known as Strong Gravitational Lensing (SGL). This has several cutting-edge applications, for example: measuring the Hubble constant and shedding more light into the apparent tension between early and late Universe, detecting the presence of massive subhalos within distant galaxies that can constrain different dark matter models, and studying a galaxy’s mass partition between baryons and dark matter with direct implications on galaxy evolution.
```

Comment:
I like it, thanks! Would replace "gravitationally induced deflections of", which I find a bit too convoluted, by "through its gravitational effect on light rays from background sources".

Also, the gravitational effect of stars is not really in the strong regime. So I would say "known as gravitational lensing", and then have an extra sentence stating what is SGL (i.e. multiple images and high distortions). What do you think?

Reply:
Ok good. Yes, feel free to edit as you like. Just to respond to your specific comments: "gravitational effect" may be too vague, so what about "gravitational deflections of"? You can indeed remove the "from vast galaxy clusters to stars within galaxies" phrase without losing much.

"Also, the gravitational effect of stars is not really in the strong regime" - you forgot about microlensing (which IS strong lensing)!! Noooooo....!!!

Reply:
Also, is it up to me to push the "Merge pull request" button below when we are happy with the edit? Should we also "Resolve the conversation"? This is nice github learning practice for me!

Reply:
Aaaah yeah ok it's semantic but indeed *micro*lensing is actually *strong* :) Ok so let's keep it this way. I actually like the "from vast clusters to stars" so let's keep it. I will add "gravitational deflection of".

PS: I think in our case, since we are all developers, anyone can "merge pull request" / "resolve conversation" stuff. I'll do it here.

Reply:
Perfect!